common/app/views/fragments/atoms/quiz.scala.html

```diff
@@ -67,41 +47,39 @@
 
 @renderAnswer(question: Question, answer: Answer, field: play.api.data.Field) = {
     <div class="quiz__answer">
-        <label for="answer-@{answer.id}" class="quiz__answer__label">
-            <input type="radio"
-                name="@{field.name}"
-                id="answer-@{answer.id}"
-                value="@{answer.id}"
-                class="quiz__answer__input">
-
-            <span class="quiz__answer__text">@{answer.text}</span>
+        <input type="radio"
+            name="@{field.name}"
+            id="answer-@{answer.id}"
+            value="@{answer.id}"
+            class="quiz__answer__input">
```

Comment:
Yes. It is always better to use more vague names because then we are not coupling it to the specific component/element and it can be nicely reused. In my experience if I have element of element I usually try to think of different way.